### DISCUSSION TITLE
How can I improve an HTML table in Open API Plugin?

### DISCUSSION THREAD

**Discussion by gii-pan** [PaloAltoNetworks/docusaurus-openapi-docs#453] [Q&A] [answered]
Hello @sserrata,

My question this time is about the table tag from HTML. 

We use Swagger 2.0, and to convert to Open API 3.0 the Swagger Editor helps a lot. 

However, the tables that we create with HTML are not working very well in the plugin as you can see: 

Web site:
![image](https://user-images.githubusercontent.com/64155751/220714344-b5a53612-d2bb-4218-aa4e-063f5991995f.png)

Code:
![image](https://user-images.githubusercontent.com/64155751/220714290-707cc4a4-804b-409b-9531-082593be3458.png)

What do you recommend? I'm fixing all of them manually, but I'd like to repair in the code and generate the tables correctly every time the documentation is update.

**Comment by sserrata** [comment, marked as answer]
Hi @gii-pan, it looks like our sanitizer regex is missing capture groups for `code`, `thead`, and `tbody`. We'll work on adding those in which should help avoid the issue you are seeing.

**Comment by sserrata** [comment]
Hi @gii-pan, just released a fix in `0.0.0-578`. Can you test and report back if it solves your issue? Thanks!

**Reply by gii-pan** [reply]
Hello @sserrata, it works \o/ 

However is happening the same with bold. 

![image](https://user-images.githubusercontent.com/64155751/223175242-0cc9036a-f299-47a2-a8dd-770fc1e9dde9.png)

**Comment by sserrata** [comment]
Ah yes, you're correct. I'll see if we can extend coverage to `bold` and remaining tags.

**Comment by sserrata** [comment]
Hi @gii-pan, can you please test with `0.0.0-588` and report if it's working?
